client/src/components/ManageBooking/BookingList/BookingGrid.tsx

```diff
@@ -0,0 +1,45 @@
+import { Grid, Avatar } from '@material-ui/core';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import SettingsIcon from '@material-ui/icons/Settings';
+import useStyles from './useStyles';
+import { useAuth } from '../../../context/useAuthContext';
+
+interface Props {
+  status: string;
+  date: string;
+}
+
+const BookingGrid = ({ status, date }: Props): JSX.Element => {
```

Comment:
This component looks really nice, though I would recommend using a different name as `BookingGrid` sounds more like a grid layout container of a sort. `BookingItem` or anything similar that sounds nice to you would be a better choice imho :).

Reply:
component renamed